### PR TITLE
[T4A5][T13-B4]Ng Huan Ran

### PR DESCRIPTION
**Pull request by shusiner**
ready for review

### CODE REVIEW

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
+package seedu.addressbook.data.tag;
+
+import java.util.*;
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
```
Comment (z11i) [inline src/seedu/addressbook/data/tag/Tagging.java:6]: Add comments about this class.

File: src/seedu/addressbook/data/tag/Tagging.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+	private Tag tag;
```
Comment (z11i) [inline src/seedu/addressbook/data/tag/Tagging.java:7]: Use 4 spaces rather than tab.

**Comment by z11i** [comment]
More changes are required in order to make the association work. Think about how existing classes and new classes can be used to utilize and associate with this Tagging class.

File: src/seedu/addressbook/data/tag/Tagging.java (hunk 2 above)
Comment (shusiner) [reply src/seedu/addressbook/data/tag/Tagging.java:7]: ok

**Pull request closed without merging**